x-pack/solutions/security/plugins/security_solution/public/common/lib/kibana/__mocks__/use_kibana.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createTGridMocks } from '@kbn/timelines-plugin/public/mock';
+
+import { createStartServicesMock } from '../kibana_react.mock';
+import { mockApm } from '../../apm/service.mock';
+import { mockCasesContract } from '@kbn/cases-plugin/public/mocks';
+
+const mockStartServicesMock = createStartServicesMock();
+
+export const useKibana = jest.fn().mockReturnValue({
+  services: {
+    ...mockStartServicesMock,
+    apm: mockApm(),
+    uiSettings: {
```

Comment:
Why these overrides over `mockStartServicesMock` ?

Reply:
The CI is not happy 😞 
I was able to fix one, but there's another that is crashing inside EUI. I am missing something, but I don't have time to dig into this rn. Rolling back to the original mock overrides. 
We can clean this as a separate task later on